pwiz_tools/Skyline/TestUtil/TestFunctional.cs

```diff
@@ -174,6 +174,22 @@ public NoNightlyTestingAttribute(string reason)
 
     }
 
+    /// <summary>
+    /// Test method attribute which specifies a test is not suitable for automated nightly leak testing
+    /// (e.g. memory hungry or excessively time consuming given that it has to be run many times for leak detection)
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = false)]
+    public sealed class NoIterativeLeakTestingAttribute : Attribute
```

Comment:
Can we give this attribute a shorter name. Maybe "NoLeakTestingAttribute"?
Also, can we give it a default zero-argument constructor which sets the Reason to "TestExclusionReason.EXCESSIVE_TIME"?

Reply:
I'm open to a rename, but I do think there's value in having to declare why it's being set.